var/spack/repos/builtin/packages/netcdf-cxx/package.py

```diff
@@ -29,17 +29,6 @@ def libs(self):
         shared = True
         return find_libraries("libnetcdf_c++", root=self.prefix, shared=shared, recursive=True)
 
-    def configure_args(self):
-        args = []
-        if "+netcdf4" in self.spec:
-            # There is no clear way to set this via configure, so set the flag
-            # explicitly
-            args.append("CPPFLAGS=-DUSE_NETCDF4")
-        # Add these to LDFLAGS explicitly, so the linker doesn't accidentally
-        # use system versions
-        ldflags = [
-            self.spec["netcdf-c"].libs.search_flags,
-            self.spec["hdf5"].libs.search_flags,
```

Comment:
@scheibelp is this maybe the reason paraview is now failing to find hdf5?

Reply:
I thought paraview uses `netcdf-c` (not `netcdf-cxx`) so I don't think changes in `netcdf-cxx` would affect it

Reply:
Ah good point, I think I'm mixing up VTK and Paraview dependencies in my head. VTK is the one that uses netcdf-cxx.